src/window/window_wrapper.rs

```diff
@@ -195,6 +199,24 @@ impl WinitWindowWrapper {
                 self.mouse_manager.enabled = mouse_enabled
             }
             WindowCommand::ListAvailableFonts => self.send_font_names(),
+            WindowCommand::Bell => {
+                let bell_command = SETTINGS.get::<WindowSettings>().bell_command;
+                let bell_command = bell_command.split(" ").collect::<Vec<_>>();
+                if bell_command.is_empty() {
+                    unsafe {
+                        // TODO: How to bell on X11/Wayland?
```

Comment:
You can change this command to "Only custom commands are supported on X11 and Wayland"

Reply:
I've changed.